tests/molecules/He_param/convert_vp_format.py

```diff
@@ -160,10 +160,10 @@ def write_to_hdf(vp, fname_out, output_complex):
     size = len(vp.name_value_pairs)
 
     f = h5py.File(fname_out,"w")
-    f.create_dataset("timestamp",data=[vp.timestamp])
+    f.create_dataset("timestamp",data=np.array([vp.timestamp],dtype=object),dtype=h5py.string_dtype('ascii'))
```

Comment:
why not utf-8?

Reply:
That would require handling UTF-8 in the C++ code that writes and reads it.  Or at least testing it that way (maybe it would work correctly already?)

Reply:
earlier in this file timestamp is decoded as utf-8.  Personally I think we should use and assume use of utf-8 encoding. We can still process our internal set strings as ascii but user supplied strings should be allowed to contain utf-8 codepoints.  In the QMCPACK code we don't seem to ever explicitly set encoding we just use datatype H5T_C_S1 which can be either ASCII or UTF-8.

Reply:
The *.vp.h5 files from QMCPACK are written with the strings as "CST H5T_CST_ASCII".  This script tries to reproduce the format of the file exactly.
If I change the script to produce 'utf8', the loader test will segfault.

Reply:
What about line 132 in this file?

Dealing with the loader segfaulting on utf-8 is beyond the scope of this PR.